airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/formats/json_spec.py

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
Could you rename this to jsonl_spec for consistency please

Reply:
done